docs/source/reference/urls.md

```diff
@@ -120,8 +120,8 @@ if the specified server were running).
 Handling this URL is the most complicated condition in JupyterHub,
 because there can be many states:
 
-1. the server is not active
-   a. user matches
+1. the server is not active/
```

Comment:
Can you clarify why have these slashes been added? I don't understand what they are meant to communicate.